src/Compilers/CSharp/Portable/Binder/BinderFactory.BinderFactoryVisitor.cs

```diff
@@ -193,6 +194,16 @@ public override Binder VisitMethodDeclaration(MethodDeclarationSyntax methodDecl
                 return resultBinder;
             }
 
+            private static Binder WithExtensionReceiverParameterBinderIfNecessary(Binder resultBinder, MethodSymbol method)
+            {
+                if (method is { IsStatic: false, ContainingType: SourceNamedTypeSymbol { IsExtension: true, ExtensionParameter: { } parameter } })
+                {
+                    return new WithParametersBinder([parameter], resultBinder);
```

Comment:
Should the doc comment of `WithParametersBinder` be updated to mention this scenario?

```cs
    /// <summary>
    /// Binder used to place the parameters of a method, property, indexer, or delegate
    /// in scope when binding &lt;param&gt; tags inside of XML documentation comments
    /// and `nameof` in certain attribute positions.
    /// </summary>
```

Reply:
> Should the doc comment of ```WithParametersBinder``` be updated to mention this scenario?

Mentioned this in a PROTOTYPE comment